static/app/utils/toArray.tsx

```diff
@@ -0,0 +1,3 @@
+export default function toArray<T>(val: T | T[]) {
+  return Array.isArray(val) ? val : [val];
+}
```

Comment:
Worth a follow-up to grep the codebase to see where else we could use this fn?

Reply:
https://github.com/getsentry/sentry/pull/41507